setup.py

```diff
@@ -270,8 +275,13 @@ def run(self):
     # See https://github.com/RaRe-Technologies/gensim/pull/2814
     # 'tensorflow',
     # 'keras',
+    'pyemd',  # see below; keep as last until appveyor issue resolved
```

Comment:
What's going on here, we don't use appveyoer any more. Merge `develop`?

Reply:
It may be more generally an issue for any windows build (which at the time of writing, was synonymous with 'appveyor'). The important thing is that it be easy to remove for the windows build, per below.

Reply:
But `linux_testenv` is defined below. This cannot be right.

I'll revert this in my PR, let's see if it's still necessary.

Reply:
It's likely still an issue with Azure Windows builds, and placing `pyemd` last makes it easy to define the windows test env as a subset of the linux one. (And perhaps, `keras` and `tensorflow` still need to be skipped on some test builds as well; I don't know.) Except to the extent such issues might prevent/break this PR's testing, they're best dealt with elsewhere by Windows-sympathizers.

Reply:
I've rearranged `setup.py` a little to make it clearer how the linux & windows package-sets relate. (Trying to enable tensorflow on py3.6 still resulted in a mysterious GC-related segfault during other doc2vec.py code, though.)